### DISCUSSION TITLE
Access previous values in afterUpdate lifecycle hook

### DISCUSSION THREAD

**Discussion by mattf96s** [strapi/strapi#8437] [Legacy Topics]
Hi

When using the afterUpdate lifecycle hook, Is it possible to access the "before update" values. This is to compare against the current values to check whether a field has changed. 

For example:
On a user collection to check whether a user has been verified and if so to send a notification.

Or will I need to create a separate collection to store this data.

Thanks

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
